src/app/modules/summary/summary.controller.js

```diff
@@ -117,6 +118,16 @@ angular
        * Go to the prizegiving view.
        */
       function prizegiving() {
+        var rounds = Match.getRounds();
+        angular.forEach(rounds, function(round) {
+          angular.forEach(getPlayers(), function(player) {
+            if (isShutout(player, rounds.indexOf(round))) {
+              PlayerStats.addShutout(player);
+            }
+          });
+        });
+        PlayerStats.updatePlayersGames();
+        PlayerStats.savePlayers();
```

Comment:
perchè fino a quel momento tutti i cambiamenti sono salvati solo sui selectedPlayers, quello aggiorna i savedPlayers che poi sono quelli che restano in memoria se ho capito bene